incubator/vault/values.yaml

```diff
@@ -56,6 +57,7 @@ vault:
     listener:
       tcp:
         address: '[::]:8200'
+        cluster_address: '[::]:8201'
```

Comment:
I can see that this cluster address has added, however it hasn't been added to the pod specification?